.github/ISSUE_TEMPLATE/10_design_proposal.md

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
If we have a design proposal template, should we also have a bug report template? This one seems to be more about proposing something new but it doesn't cover the case of "this isn't working right" as well.

Reply:
Added a bug report template.